src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -499,6 +499,10 @@ def _iterator(self, result: typing.Any) -> Iterator:
         while iterator.hasNext():
             yield iterator.next()
 
+    @staticmethod
+    def _option_as_python(scala_option: typing.Any):
```

Comment:
Missing return type? Or we do not have this for scala objects?

Reply:
These are magic proxy objects, indeed, that bridge to JVM objects. I'd add the type but haven't been able to figure out (yet) what it is. In some ways they're the same as mocks: anything goes. Unlike mocks though, we definitely _don't_ have a type that represents the interface of the target JVM objects:

 - `scala_option` is: `scala.Option[T]`
 - The return type is: `T`